website/project/views/node.py

```diff
@@ -993,6 +999,7 @@ def node_child_tree(user, node_ids):
             'category': node.category,
             'permissions': {
                 'view': can_read,
+                'is_admin': node.has_permission(user, 'read')
```

Comment:
Minor: There's redundancy and confusing nesting here. There's `item['node']['is_admin']` and `item['permissions']['is_admin']`. "Write" permissions are under `item['node']` but read permissions are under `item['permissions']['view']`.

That said, this code was already in place. Feel free to refactor this in a followup PR.